lib/railway_ipc.rb

```diff
@@ -16,8 +17,11 @@
 require 'railway_ipc/responder'
 require 'railway_ipc/rpc/rpc'
 require 'railway_ipc/consumer/consumer'
+require 'railway_ipc/models/published_message'
 require 'railway_ipc/railtie' if defined?(Rails)
 
+Sneakers.configure
```

Comment:
Some logging was failing without this. This sets a default sneakers logger. Otherwise, it's `nil` and that breaks all the things. May be a result of the upgrade.

Reply:
We are configuring the sneaker's logger when we configure sneakers in Ironboard. Was it failing when included in Ironboard too? Wondering if we need to change the way we configure it there. https://github.com/flatiron-labs/ironboard/blob/master/config/initializers/sneakers.rb